src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java

```diff
@@ -147,6 +147,18 @@ public void updatePOTeamSelection(String caseUUIDString, String stageUUIDString,
         log.debug("######## Updated Team Selection at PO ########");
     }
 
+    public void updateValue(String caseUUIDString, String stageUUIDString, String key, String value) {
+        UUID caseUUID = UUID.fromString(caseUUIDString);
+        UUID stageUUID = UUID.fromString(stageUUIDString);
+        log.info("Update {} key to {} value", key, value);
+        if (StringUtils.hasText(key)){
+            Map<String, String> updateValue = new HashMap<>();
+            updateValue.put(key, value);
```

Comment:
Map.of(Key, Value) would be neater here :)

Reply:
Improvement implemented.